packages/climate-ref/src/climate_ref/executor/hpc.py

```diff
@@ -36,6 +37,78 @@
 from .local import ExecutionFuture, process_result
 
 
+class SlurmConfig(BaseModel):
+    """Slurm Config Pydnatic model"""
```

Comment:
```suggestion
    """Slurm Config Pydantic model"""
```

This Pydantic-part is an implementation detail so I'm not sure if its needed

Reply:
Yes. I will remove it.